DEVELOPERS.md

```diff
@@ -37,12 +39,54 @@ _Note:_ you will need the [Bitwarden CLI tool](https://bitwarden.com/help/articl
 - **Pip**
 - **Node.js v16.x** ([fnm](https://github.com/Schniz/fnm#installation) is recommended)
 - **npm v7.x**
+- **Postgres**
 
 #### `just` commands
 
 Each `just` command sets up a dev environment as part of running it.
 If you want to maintain your own virtualenv make sure you have activated it before running a `just` command and it will be used instead.
 
+
+#### Postgres
+
+##### Installing on macOS
+
+[Postgres.app](https://postgresapp.com/) is the easiest way to run Postgres on macOS, you can install it from homebrew (casks) with:
+
+  brew install --cask postgres-unofficial
+
+You will need to add its bin directory to your path for the CLI tools to work.
+
+Postico is a popular GUI for Postgres:
+
+  brew install --cask postico
+
+
+##### Installing on Linux
+
+Install postgresql with your package manager of choice.
+
+[This guide](http://meshy.co.uk/posts/postgresql-without-passwords) explains how to set up ident (password-less) auth, and set some options for faster, but more dangerous, performance.
+
+If you need to upgrade an installation then the Arch Wiki has the [best instructions](https://wiki.archlinux.org/title/PostgreSQL#Upgrading_PostgreSQL) I've found.
+
+
+##### Creating a database
+
+You'll need a database in Postgres to work with, run:
+
+  psql -c "create database jobserver"
+
+
+##### Restoring Backups
+
+Backups of production can be restored to a local database using this command:
+
+  pg_restore --no-acl --no-owner -d jobserver jobserver.dump
```

Comment:
you might also want `--clean` which will wipe a pre-existing database (if it exists)?

Reply:
This worked better than expected, you don't even have to quit all your connections 🤩